### PR TITLE
Update explain and execution summary related docs

### PR DESCRIPTION
**Pull request by yibin87**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v8.5 (TiDB 8.5 versions)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by windtalker — APPROVED** [review]
lgtm

File: explain-overview.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ Records: 2  Duplicates: 0  Warnings: 0
 
 TiDB 会汇聚 TiKV/TiFlash 上扫描的数据或者计算结果，这种“数据汇聚”算子目前有如下几类：
 
-- **TableReader**：将 TiKV 上底层扫表算子 TableFullScan 或 TableRangeScan 得到的数据进行汇总。
+- **TableReader**：将 TiKV/TiFlash 上底层算子得到的数据进行汇总。
```
Comment (hfxsd) [inline explain-overview.md:148]:
```suggestion
- **TableReader**：将 TiKV 和 TiFlash 上底层算子得到的数据进行汇总。
```

File: explain-overview.md [hunk 2]
```diff
@@ -174,9 +174,9 @@ Build 总是先于 Probe 执行，并且 Build 总是出现在 Probe 前面。
 
 ### Task 简介
 
-目前 TiDB 的计算任务分为两种不同的 task：cop task 和 root task。Cop task 是指使用 TiKV 中的 Coprocessor 执行的计算任务，root task 是指在 TiDB 中执行的计算任务。
+目前 TiDB 的计算任务分为四种不同的 task：root task, cop task, batchCop task 和 mpp task。Root task 是指在 TiDB 中执行的计算任务；cop task 是指使用 TiKV/TiFlash 中的 Coprocessor 执行的计算任务；batchCop task 是对 TiFlash cop task 的一种优化，可以在一个任务中执行对多个 region 的查询；mpp task 是指利用 TiFlash 的 [MPP 模式](/explain-mpp.md)执行查询。
```
Comment (hfxsd) [inline explain-overview.md:177]:
```suggestion
目前 TiDB 的计算任务分为四种不同的 task：root task, cop task, batchCop task 和 mpp task。

- root task 是指在 TiDB 中执行的计算任务；
- cop task 是指使用 TiKV 和 TiFlash 中的 Coprocessor 执行的计算任务；
- batchCop task 是对 TiFlash cop task 的一种优化，可以在一个任务中执行对多个 Region 的查询；
- mpp task 是指利用 TiFlash 的 [MPP 模式](/explain-mpp.md)执行查询。
```

File: explain-overview.md [hunk 3]
```diff
@@ -174,9 +174,9 @@ Build 总是先于 Probe 执行，并且 Build 总是出现在 Probe 前面。
...
 
-SQL 优化的目标之一是将计算尽可能地下推到 TiKV 中执行。TiKV 中的 Coprocessor 能支持大部分 SQL 内建函数（包括聚合函数和标量函数）、SQL `LIMIT` 操作、索引扫描和表扫描。
+SQL 优化的目标之一是将计算尽可能地下推到 TiKV/TiFlash 中执行。TiKV 中的 Coprocessor 能支持大部分 SQL 内建函数（包括聚合函数和标量函数）、SQL `LIMIT` 操作、索引扫描和表扫描。TiFlash 中的 Coprocessor 与 TiKV 功能类似，不过不支持索引扫描。
```
Comment (hfxsd) [inline explain-overview.md:179]:
```suggestion
SQL 优化的目标之一是将计算尽可能地下推到 TiKV 或 TiFlash 中执行。TiKV 中的 Coprocessor 能支持大部分 SQL 内建函数（包括聚合函数和标量函数）、SQL `LIMIT` 操作、索引扫描和表扫描。TiFlash 中的 Coprocessor 与 TiKV 功能类似，但是不支持索引扫描。
```

File: sql-statements/sql-statement-explain-analyze.md [hunk 4]
```diff
@@ -39,9 +39,9 @@ ExplainableStmt ::=
 | 属性名          | 含义 |
 |:----------------|:---------------------------------|
 | actRows       | 算子实际输出的数据条数。 |
-| execution info  | 算子的实际执行信息。time 表示从进入算子到离开算子的全部 wall time，包括所有子算子操作的全部执行时间。如果该算子被父算子多次调用 (loops)，这个时间就是累积的时间。loops 是当前算子被父算子调用的次数。 |
-| memory  | 算子占用内存空间的大小。 |
-| disk  | 算子占用磁盘空间的大小。 |
+| execution info  | 算子的实际执行信息。time 表示从进入算子到离开算子的全部 wall time，包括所有子算子操作的全部执行时间。如果该算子被父算子多次调用 (loops)，这个时间就是累积的时间。open 表示算子初始化所花费的时间。close 表示从算子处理完所有数据到算子结束的时间。其中 time 统计的时间是包含 open 和 close 的时间。当算子存在多并发执行情况时，执行信息中显示的会是各个并发使用的 wall time 加总求和的结果，此时 time, open 和 close 会被替换为 total_time, total_open 和 total_close。loops 是当前算子被父算子调用的次数。 |
+| memory  | 算子占用的最大内存空间。 |
+| disk  | 算子占用的最大磁盘空间大小。 |
```
Comment (hfxsd) [inline sql-statements/sql-statement-explain-analyze.md:44]:
```suggestion
| `execution info`  | 算子的实际执行信息。`time `表示从进入算子到离开算子的全部` wall time`，包括所有子算子操作的全部执行时间。如果该算子被父算子多次调用 (`loops`)，这个时间就是累积的时间。`loops `是当前算子被父算子调用的次数。`open` 表示算子初始化所需的时间。`close` 表示从算子处理完所有数据到算子结束的时间。其中 `time` 统计的时间是包含 `open` 和 `close` 的时间。当算子存在多并发执行情况时，执行信息中显示的会是各个并发使用的 `wall time` 加总求和的结果，此时 `time`、`open` 和 `close` 会被替换为 `total_time`, `total_open` 和 `total_close`。|
| `memory`  | 算子占用的最大内存空间。 |
| `disk`  | 算子占用的最大磁盘空间。 |
```

File: sql-statements/sql-statement-explain-analyze.md [hunk 5]
```diff
@@ -39,9 +39,9 @@ ExplainableStmt ::=
 | 属性名          | 含义 |
 |:----------------|:---------------------------------|
 | actRows       | 算子实际输出的数据条数。 |
```
Comment (hfxsd) [inline sql-statements/sql-statement-explain-analyze.md:41]:
```suggestion
| `actRows`       | 算子实际输出的数据条数。 |
```

File: sql-statements/sql-statement-explain-analyze.md [hunk 6]
```diff
@@ -339,6 +339,18 @@ after key/value request is processed:
 
 对于 writes 和 batch gets，计算方法相似，只是基础成本不同。
 
+### tiflash_wait 信息
+
+当查询涉及到 MPP task 时候时，执行时间还受到各类 tiflash_wait 时间的影响，示例如下：
```
Comment (hfxsd) [inline sql-statements/sql-statement-explain-analyze.md:344]:
```suggestion
当查询涉及到 mpp task 时候时，执行时间还受到各类 `tiflash_wait` 时间的影响，示例如下：
```

File: sql-statements/sql-statement-explain.md [hunk 7]
```diff
@@ -52,7 +52,7 @@ ExplainableStmt ::=
 |:----------------|:----------------------------------------------------------------------------------------------------------|
 | id            | 算子的 ID，是算子在整个执行计划中唯一的标识。在 TiDB 2.1 中，ID 会格式化地显示算子的树状结构。数据从孩子结点流向父亲结点，每个算子的父亲结点有且仅有一个。|
 | estRows       | 算子预计将会输出的数据条数，基于统计信息以及算子的执行逻辑估算而来。在 4.0 之前叫 count。 |
-| task          | 算子属于的 task 种类。目前的执行计划分成为两种 task，一种叫 **root** task，在 tidb-server 上执行，一种叫 **cop** task，在 TiKV 或者 TiFlash 上并行执行。当前的执行计划在 task 级别的拓扑关系是一个 root task 后面可以跟许多 cop task，root task 使用 cop task 的输出结果作为输入。cop task 中执行的也即是 TiDB 下推到 TiKV 或者 TiFlash 上的任务，每个 cop task 分散在 TiKV 或者 TiFlash 集群中，由多个进程共同执行。 |
+| task          | 算子属于的 task 种类。目前的执行计划可以分成为四种 task：第一种叫 **root** task，在 tidb-server 上执行；第二种叫 **cop** task，在 TiKV 或者 TiFlash 上并行执行；第三种叫 **batchCop** task，在 TiFlash 上并行执行；第三种叫 **mpp** task，在 TiFlash 上并行执行。当前的执行计划在 task 级别的拓扑关系是一个 root task 后面可以跟许多其他种类 task，root task 使用其他种类 task 的输出结果作为输入。其他种类 task 中执行的也即是 TiDB 下推到 TiKV 或者 TiFlash 上的任务，每个 task 分散在 TiKV 或者 TiFlash 集群中，由多个进程共同执行。 |
```
Comment (hfxsd) [inline sql-statements/sql-statement-explain.md:55]:
```suggestion
| task          | 算子属于的 task 种类。目前的执行计划可以分为以下四种 task：**root** task，在 tidb-server 上执行；**cop** task，在 TiKV 或者 TiFlash 上并行执行；**batchCop** task，在 TiFlash 上并行执行；**mpp** task，在 TiFlash 上并行执行。当前的执行计划在 task 级别的拓扑关系是一个 root task 后面可以跟许多其他种类的 task，root task 使用其他种类 task 的输出结果作为输入。其他种类 task 中执行的也即是 TiDB 下推到 TiKV 或者 TiFlash 上的任务，每个 task 分散在 TiKV 或者 TiFlash 集群中，由多个进程共同执行。 |
```

File: explain-overview.md [hunk 8]
```diff
@@ -145,7 +145,7 @@ Records: 2  Duplicates: 0  Warnings: 0
 
 TiDB 会汇聚 TiKV/TiFlash 上扫描的数据或者计算结果，这种“数据汇聚”算子目前有如下几类：
 
-- **TableReader**：将 TiKV 上底层扫表算子 TableFullScan 或 TableRangeScan 得到的数据进行汇总。
+- **TableReader**：将 TiKV 和 TiFlash 上底层算子得到的数据进行汇总。
```
Comment (yibin87) [inline explain-overview.md:148]:
```suggestion
- **TableReader**：将 TiKV 或 TiFlash 上底层算子得到的数据进行汇总。
```

File: explain-overview.md [hunk 9]
```diff
@@ -174,9 +174,14 @@ Build 总是先于 Probe 执行，并且 Build 总是出现在 Probe 前面。
 
 ### Task 简介
 
-目前 TiDB 的计算任务分为两种不同的 task：cop task 和 root task。Cop task 是指使用 TiKV 中的 Coprocessor 执行的计算任务，root task 是指在 TiDB 中执行的计算任务。
+目前 TiDB 的计算任务分为四种不同的 task：root task, cop task, batchCop task 和 mpp task。
 
-SQL 优化的目标之一是将计算尽可能地下推到 TiKV 中执行。TiKV 中的 Coprocessor 能支持大部分 SQL 内建函数（包括聚合函数和标量函数）、SQL `LIMIT` 操作、索引扫描和表扫描。
+- root task 是指在 TiDB 中执行的计算任务；
+- cop task 是指使用 TiKV 和 TiFlash 中的 Coprocessor 执行的计算任务；
```
Comment (yibin87) [inline explain-overview.md:180]:
```suggestion
- cop task 是指使用 TiKV 或 TiFlash 中的 Coprocessor 执行的计算任务；
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: explain-overview.md [hunk 10]
```diff
@@ -174,9 +174,14 @@ Build 总是先于 Probe 执行，并且 Build 总是出现在 Probe 前面。
 
 ### Task 简介
 
-目前 TiDB 的计算任务分为两种不同的 task：cop task 和 root task。Cop task 是指使用 TiKV 中的 Coprocessor 执行的计算任务，root task 是指在 TiDB 中执行的计算任务。
+目前 TiDB 的计算任务分为四种不同的 task：root task, cop task, batchCop task 和 mpp task。
```
Comment (qiancai) [inline explain-overview.md:177]:
```suggestion
目前 TiDB 的计算任务分为四种不同的 task：root task, cop task, batchCop task 和 MPP task。
```

**Review by hfxsd — APPROVED** [review]

**Pull request merged**